code/game/machinery/buttons.dm

```diff
@@ -47,4 +47,50 @@
 	var/id = 1
 
 	attack_paw(mob/user as mob)
-		return
\ No newline at end of file
+		return
+
+/obj/machinery/medical_help_button
+	name = "Medical attention required"
+	icon = 'icons/obj/stationobjs.dmi'
+	icon_state = "doorctrl0"
+	desc = "A button for alerting doctors that you require assistance."
+	var/id = null
+	var/active = FALSE
+	anchored = 1.0
+	use_power = 1
+	idle_power_usage = 2
+	active_power_usage = 4
+
+/obj/machinery/medical_help_button/attack_hand(mob/living/carbon/human/user)
+	add_fingerprint(user)
```

Comment:
This should likely be below the istype. (as if they can't use it anyway what use is to know for us that they touched it? Do xenos even have fingerprints?)